docs/about/CONTRIBUTORS.md

```diff
@@ -1,10 +1,17 @@
 # Contributors
 
-* [Stephanie (Stimac) Drescher ![image](https://avatars1.githubusercontent.com/u/18073131?v=3&s=100)](https://github.com/ststimac)
-* [David García ![image](https://avatars1.githubusercontent.com/u/1581288?v=3&s=100)](https://github.com/sarvaje)
-* [Catalin Maris ![image](https://avatars2.githubusercontent.com/u/1223565?v=3&s=100)](https://github.com/alrra)
-* [Antón Molleda ![image](https://avatars2.githubusercontent.com/u/606594?v=3&s=100)](https://github.com/molant)
-* [Qing Zhou ![image](https://avatars3.githubusercontent.com/u/20218531?v=3&s=100)](https://github.com/qzhou1607)
+<!-- Contributors START
+Stephanie_Stimac_Drescher ststimac https://github.com/ststimac code design
+David_Garcia sarvaje https://github.com/sarvaje code
+Catalin_Maris alrra https://github.com/alrra code
+Antón_Molleda molant https://github.com/molant code
+Qing_Zhou qzhou1607 https://github.com/qzhou1607 code
+Contributors END -->
+<!-- Contributors table START -->
+| [<img src="https://avatars.githubusercontent.com/ststimac?s=100" width="100" alt="Stephanie Stimac Drescher" /><br /><sub>Stephanie Stimac Drescher</sub>](https://github.com/ststimac)<br />💻 🎨 | [<img src="https://avatars.githubusercontent.com/sarvaje?s=100" width="100" alt="David Garcia" /><br /><sub>David Garcia</sub>](https://github.com/sarvaje)<br />💻 | [<img src="https://avatars.githubusercontent.com/alrra?s=100" width="100" alt="Catalin Maris" /><br /><sub>Catalin Maris</sub>](https://github.com/alrra)<br />💻 | [<img src="https://avatars.githubusercontent.com/molant?s=100" width="100" alt="Antón Molleda" /><br /><sub>Antón Molleda</sub>](https://github.com/molant)<br />💻 | [<img src="https://avatars.githubusercontent.com/qzhou1607?s=100" width="100" alt="Qing Zhou" /><br /><sub>Qing Zhou</sub>](https://github.com/qzhou1607)<br />💻 |
```

Comment:
* Use `<br>` instead of `<br />`.
* Garcia => García (@sarvaje?)
* Catalin Maris => Cătălin Mariș